astro.config.mjs

```diff
@@ -82,6 +82,7 @@ export default defineConfig({
 						{ label: 'Adding a new Astro + X Guide', link: '/guides/new-third-party-stub/' },
 						{ label: 'Contributing a Recipe', link: '/guides/recipe-writing/' },
 						{ label: 'Translating Astro docs', link: '/guides/i18n/' },	
+						{ label: 'Contributing on Hacktoberfest', link: '/guides/hacktoberfest/' },
```

Comment:
My first thought is this is pretty buried on the page, so let's have a banner (OK to add with this PR and have shown as soon as it merges.)

Reply:
I assume you mean a banner on the homepage? I'll add one and then you can reword/change places if you want!